app/models/nic/interface.rb

```diff
@@ -54,6 +55,18 @@ def fqdn_was
 
     protected
 
+    def identifier_change
+      return true if self.new_record?
```

Comment:
I think you can move up this line to the validation and say 
`validate :identifier_change, :unless => -> { |interface| interface.new_record? } `